app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditViewModel.kt

```diff
@@ -1092,7 +1095,9 @@ class VaultAddEditViewModel @Inject constructor(
         updateLoginContent { loginType ->
             loginType.copy(
                 uriList = loginType.uriList + UriItem(
-                    id = UUID.randomUUID().toString(),
+                    id = UUID
+                        .randomUUID()
+                        .toString(),
```

Comment:
Any reason to make this change?

Reply:
Hmm, that should not happen as part of the formatter. To be clear, there is nothing wrong with this change but the regular auto-formatter does not normally do this. 🤔